optimizer/optimize-spec.k.tmpl

```diff
@@ -1,9 +1,10 @@
-requires "lemmas/mcd/verification.k"
+requires "lemmas/infinite-gas.k"
 
 module OPTIMIZE
     imports INT
-    imports LEMMAS-MCD
```

Comment:
Why was `LEMMAS-MCD` imported and why is it no longer needed ?

Reply:
It was imported so we could get that body of lemmas for doing these optimizations. It's a large body of lemmas that was developed for the maker proofs.

But I realized those lemmas weren't needed for these proofs, and the smaller the trust base we can have, the better.